samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -167,11 +168,26 @@ object SamzaContainer extends Logging {
       }
     }
 
+    val changeLogSystemStreams = config
+      .getStoreNames
+      .filter(config.getChangelogStream(_).isDefined)
+      .map(name => (name, config.getChangelogStream(name).get)).toMap
+      .mapValues(StreamUtil.getSystemStreamFromNames(_))
+
+    info("Got change log system streams: %s" format changeLogSystemStreams)
+
+    // In case of standby-tasks, we set its inputSystems to be the changeLogSystems so that the
```

Comment:
Minor: I think it'll be better to 
1. Do this as two separate collections and merge them. (val inputSSPs = ..., val standBySSPs = ..., val taskSSPs = inputSSPs ++ standBySSPs). Variable names are just suggestions.
2. Clarify this comment. I understood what it means but might not be obvious to someone who doesn't have enough context.